src/System.Reflection.DispatchProxy/ref/System.Reflection.DispatchProxy.csproj

```diff
@@ -3,10 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{7DF3C428-AAD6-41C7-98E6-6CACFD5C391E}</ProjectGuid>
+    <PackageTargetFramework>netstandard2.0;uap10.0.16299</PackageTargetFramework>
```

Comment:
Shouldn't the NS2.0 version apply or are we *still* missing a NuGet.Frameworks update?

Reply:
I believe we are still missing an update of the mappings.

Reply:
I would expect this mapping to exist as well.